rasa/core/brokers/sql.py

```diff
@@ -5,6 +5,8 @@
 from typing import Any, Dict, Optional, Text, Generator
 
 from sqlalchemy.orm import Session
+from sqlalchemy.ext.declarative import declarative_base, DeclarativeMeta
+from sqlalchemy import Column, Integer, String, Text
```

Comment:
@wochinge I've renamed this `Text` because it clashed with `typing.Text`. I hope it's okay this way.

Reply:
Sure 👍🏻 I think using the full module is bit preferable to renaming (e.g. referring to it as `sqlachemy.Text`) but fine for now